index.html

```diff
@@ -1,114 +1,125 @@
-                                <p class="detail-card">15 MPH</p>
+                                <p id="currWindSpeed" class="detail-card">15 MPH</p>
                                 <p class="detail-title">WIND SPEED</p>
                             </div>
-                            </div>
                         </div>
                     </div>
                 </div>
-                <div class="uk-card uk-card-default uk-card-body uk-width-1-2@m weather-card">
-                    <div class="weekly-card-header">
-                        WEEKLY FORECAST
-                    </div>
-                    <div class="weather-forecast-weekly">
-                        <div class="uk-grid-small uk-child-width-expand" uk-grid>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; /  45&deg;</p>
-                                    <p class="weekly-detail-title">WED</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">THURS</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">FRI</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">SAT</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">SUN</p>
-                                </div>
-                                <div class="uk-card uk-card-default uk-card-hover uk-card-body uk-padding-remove-horizontal">
-                                    <p class="weekly-detail-card">28&deg; / 45&deg;</p>
-                                    <p class="weekly-detail-title">MON</p>
-                                </div>
-                            </div>
-                            </div>
-                        </div>
-                    </div>
+            </div>
 
+        </div>
+    </div>
+
```

Comment:
please remove lines 73 & 74... this appears to be where the formatting is getting messed up. We didn't add divs to this layout with this commit, so there shouldn't need to be two additional closing div tags. If I'm missing something here, let me know.